package.json

```diff
@@ -1,18 +1,21 @@
 {
-  "name": "findadoc",
+  "name": "@ourjapanlife/findadoc-frontend",
   "version": "1.0.0",
   "private": true,
+  "license": "BSD-3",
   "scripts": {
     "dev": "nuxt dev",
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate",
-    "netlify": "git submodule update --init --recursive --remote && nuxt generate",
+    "prenetlify": "echo //npm.pkg.github.com/:_authToken=$GITHUB_ACCESS_TOKEN >> ~/.yarnrc",
```

Comment:
Apparently one has to edit the user level npmrc not the local one 

https://github.community/t/netlify-getting-401-from-github-package-registry-with-auth-token/16415/3